### PR TITLE
Camel 13480

### PR DESCRIPTION
**Pull request by akoufoudakis**
PLEASE REVIEW
This are the sample files for the issue https://issues.apache.org/jira/browse/CAMEL-13480.
All the components, starting with "a"

### CODE REVIEW

File: components/camel-ahc/src/main/docs/ahc-component.adoc [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 *Available as of Camel version 2.8*
 
-The ahc: component provides HTTP based endpoints
+The *ahc:* component provides HTTP based endpoints
 for consuming external HTTP resources (as a client to call external
```
Comment (oscerd) [inline components/camel-ahc/src/main/docs/ahc-component.adoc]: The *ahc:* is not rendered well on the website. 

https://camel.apache.org/staging/

Comment (akoufoudakis) [reply components/camel-ahc/src/main/docs/ahc-component.adoc]: Changed to capitals.

**Comment by davsclaus** [comment]
There is some merge conflicts. I wonder if you would try to resolve them on your end and push again

**Comment by akoufoudakis** [comment]
> There is some merge conflicts. I wonder if you would try to resolve them on your end and push again

Sure. Will do it later today.

**Comment by davsclaus** [comment]
Sorry this PR seems to get broken and there are now alot of files - this can happen sometimes with git. I suggest to switch to master, do a pull so you get latest changes, and then create a new branch, and cherry-pick the commits from this PR with only YOUR changes. And then do another PR, and close this one.

**Comment by davsclaus** [comment]
@akoufoudakis as suggested can you work on a new PR and try again. We are closing this PR in the mean time.

**Pull request closed without merging**